src/account.ts

```diff
@@ -192,3 +273,21 @@ export const importPublic = function(publicKey: Buffer): Buffer {
   }
   return publicKey
 }
+
+/**
+ * Returns a zero address.
+ */
```

Comment:
Can we change "a" to "the" here for enhanced clarity?

Reply:
fixed in https://github.com/ethereumjs/ethereumjs-util/pull/275/commits/e48478d3449fed6142cd6f611dffab5d4ed46ee3